### ISSUE TITLE
Tutorial for updating ModelSEED

### ISSUE THREAD

**Issue by samseaver** [ModelSEED/ModelSEED#154]
So I just tried `ms update`.  I don't think I've ever used it before, as all this time, I've just been doing `git pull` for the master branch.  But for OS X, I got this:

```
Cannot update your system automatically
Sorry, your os is not supported
```

This is fine, but it's not very helpful.  There doesn't seem to be anything in the wiki about this, so I think there should be a tutorial page for _Updating_ ModelSEED, especially if one were to use git and/or carton manually.

**Comment by pfrybar** [comment]
Yeah, as it stands right now you can only run the update command if you used the install script. It sets some config variables in a perl package that helps it update correctly. I'll probably change this down the road once I have a little more time.

**Comment by samseaver** [comment]
It's been a while since I used that script, have you updated it since?

**Comment by cshenry** [comment]
Seems like this should be generalized to always running git pull and the carton install command (or equivalent).
